src/autoscan_inotify.cc

```diff
@@ -50,7 +50,7 @@ AutoscanInotify::AutoscanInotify(std::shared_ptr<Storage> storage, std::shared_p
     std::error_code ec;
     if (isRegularFile(INOTIFY_MAX_USER_WATCHES_FILE, ec)) {
         try {
-            int max_watches = std::stoi(trim_string(readTextFile(INOTIFY_MAX_USER_WATCHES_FILE)));
+            [[maybe_unused]] int max_watches = std::stoi(trim_string(readTextFile(INOTIFY_MAX_USER_WATCHES_FILE)));
```

Comment:
Personally I would say that we should promote this message to an INFO as it can explain why some changes are not noticed by inotify (hit the watch limit)

Reply:
Just printing this (at any level) doesn't give user more information than can be just `cat` ing this file.

More user-friendly version is to do this when the limit is hit, e.g. as vs code does: when it can't create more watches it prints a message explaining what happened and how to tune this limit.